dojo.py

```diff
@@ -97,11 +97,122 @@ def allocate_room(self, new_person, room):
             living_space.occupants.append(new_person)
             print('{0} has been allocated the living spaces {1}'.format(new_person.person_name, living_space.room_name))
 
-    def print_room(self, room_name):
-        pass
+    def print_room(self, room):
+        """ checks if room name given exists in the list of total rooms.
+            if so it prints out a statement that highlights the room name,
+            the room type and occupants.
+        """
+        if self.total_rooms:
+            if room in [room.room_name for room in self.total_rooms]:
```

Comment:
what does this line implement. I think it may be redundant with what line 107 and 108 do